obs_status.yml

```diff
@@ -0,0 +1,2147 @@
+  mp_starcat_time: 2018:155:22:59:34.754
+  status: 1
+- agasc_id:
+  - 738075032
+  comments: Spoiler 30 arcsec away
+  mp_starcat_time: 2018:158:10:17:36.762
+  status: 1
+- agasc_id:
+  - 49818760
+  comments: Faint star. Affected by imposter. Set mag to MAXMAG
+  mp_starcat_time: 2018:165:06:34:21.353
+  status: 1
+- agasc_id:
+  - 745014832
+  comments: Tracking affected by imposter
+  mp_starcat_time: 2018:198:01:56:54.706
+  status: 1
+- agasc_id:
+  - 974925328
+  comments: Faint star. Tracking affected by imposters. Marginally acceptable
+  mp_starcat_time: 2018:214:12:04:13.567
+  status: 0
+- agasc_id:
+  - 8395104
+  comments: Magnitude changes within observation. Eclipsing binary
+  mp_starcat_time: 2018:295:05:40:26.899
+  status: 1
+- agasc_id:
+  - 168038608
+  comments: Faint star. Tracking affected by imposters. Marginally acceptable
+  mp_starcat_time: 2018:330:13:00:19.280
+  status: 0
+- agasc_id:
+  - 350392600
+  comments: Bright star. Never acquired. Set mag to 5.0
+  mp_starcat_time: 2019:012:01:07:46.603
+  status: 1
+- agasc_id:
+  - 109061768
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2019:035:04:23:05.219
+  status: 1
+- agasc_id:
+  - 989727248
+  comments: UNKNOWN
+  mp_starcat_time: 2019:092:13:40:47.000
+  status: 1
+- agasc_id:
+  - 188501784
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2019:113:10:12:41.647
+  status: 1
+- agasc_id:
+  - 120457376
+  comments: Faint star affected by imposters. Set magnitude to MAXMAG
+  mp_starcat_time: 2019:121:21:47:55.999
+  status: 1
+- agasc_id:
+  - 1105998760
+  comments: Star gets 2 mags dimmer
+  mp_starcat_time: 2019:190:08:02:57.107
+  status: 1
+- agasc_id:
+  - 836381648
+  comments: Faint star. Affected by imposters. Set mag to MAXMAG
+  mp_starcat_time: 2019:245:18:05:01.870
+  status: 1
+- agasc_id:
+  - 157951008
+  comments: Faint star. Tracking affected by imposters. Marginally acceptable
+  mp_starcat_time: 2019:304:07:33:22.655
+  status: 0
+- agasc_id:
+  - 166993856
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2019:336:01:10:48.142
+  status: 1
+- agasc_id:
+  - 1178221128
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2019:336:03:32:40.722
+  status: 1
+- agasc_id:
+  - 122159368
+  comments: Affected by imposters
+  mp_starcat_time: 2019:357:18:41:21.840
+  status: 0
+- agasc_id:
+  - 1108353064
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2020:026:14:02:58.573
+  status: 1
+- agasc_id:
+  - 33690776
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2020:045:02:16:37.894
+  status: 1
+- agasc_id:
+  - 110493968
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2020:055:11:57:17.800
+  status: 1
+- agasc_id:
+  - 33690776
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2020:057:21:03:53.096
+  status: 1
+- agasc_id:
+  - 111028544
```

Comment:
This is one of a handful of stars already in the bad table.  Not sure if we care.

Reply:
We do. I left a comment below (reading from bottom to top)